specs.wdio/helpers/session.ts

```
@@ -2,7 +2,9 @@
 /* eslint-disable no-eval */
 import { multiremote } from 'webdriverio';
 import WdioRemote from './wdio_remote';
-import AttributesRepository from '../../src/browser_sdk/attributes_repository';
+import RecordsRepository from '../../src/browser_sdk/records_repository';
+
+type AttributesRepository = RecordsRepository;
 import FactsRepository from '../../src/browser_sdk/facts_repository';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)